.github/scripts/test.sh

```diff
@@ -0,0 +1,2 @@
+flake8 --exit-zero --statistics --config build_tools/flake8.cfg RefRed
```

Comment:
Where is `flake8.cfg`? I can't find it.

Reply:
I couldnt tell you without some looking myself as this is direct carry over from the `.travis.yml` file.  Perhaps @mdoucet remembers?   looks like it was added as part of [this commit](https://github.com/neutrons/RefRed/commit/b4be97401a32b983e61aa38a7a60ae3ecd300ae6#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485) originally

Reply:
Looks like it was removed on `next` but I found it is still in `main` https://github.com/neutrons/RefRed/blob/main/build_tools/flake8.cfg

I will bring it back.

Reply:
Actually, this config got moved into `setup.cfg` so it shouldn't be needed to be specified. I will update the command to remove `--config build_tools/flake8.cfg`.

Reply:
yeah, I removed `flake.cfg` to be consistent with other repos where we're putting this info (and other) under `setup.cfg`. I'm sorry I didn't catch the existing usages of `flake.cfg`